app/javascript/components/Card.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import 'stylesheets/card';
+
+const Card = ({ children, className }) => <div className={`card ${className}`}>{children}</div>;
```

Comment:
This component implement the white card with drop shadow style that we use in many places on the website.

<img width="341" alt="Screen Shot 2022-01-24 at 4 49 39 PM" src="https://user-images.githubusercontent.com/9601737/150870698-bc405036-f0a4-41c6-ae07-592f5da13c50.png">
s